### DISCUSSION TITLE
Story Agent项目https://www.modelscope.cn/studios/iic/story_agent/summary

### DISCUSSION THREAD

**Discussion by cgwyx** [modelscope/ms-agent#375] [Q&A]
Story Agent项目https://www.modelscope.cn/studios/iic/story_agent/summary， 文生图AI-ModelScope/stable-diffusion-xl-base-1.0模型已下载到本地，如何实现本地调用_local_call，
需要修改这个文件配置\modelscope-agent\demo\story_agent\cfg_tool_template.json吗？

**Comment by zzhangpurdue** [comment]
story agent这个分支有点老，接口变化很大，不过你可以通过master上的local_call的demo尝试应用在这个分支上，新建一个tool。
